tests/commands/install_uninstall_test.py

```diff
@@ -759,6 +759,45 @@ def test_post_commit_integration(tempdir_factory, store):
         assert os.path.exists('post-commit.tmp')
 
 
+def test_post_merge_integration(tempdir_factory, store):
+    path = git_dir(tempdir_factory)
+    config = [
+        {
+            'repo': 'local',
+            'hooks': [{
+                'id': 'post-merge',
+                'name': 'Post merge',
+                'entry': 'touch post-merge.tmp',
+                'language': 'system',
+                'always_run': True,
+                'verbose': True,
+                'stages': ['post-merge'],
+            }],
+        },
+    ]
+    write_config(path, config)
+    with cwd(path):
+        #  create a simple diamond of commits for a non-trivial merge
+        open('init', 'a').close()
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        open('master', 'a').close()
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        cmd_output('git', 'checkout', '-b', 'branch', 'HEAD^')
+        open('branch', 'a').close()
+        cmd_output('git', 'add', '.')
+        git_commit()
+
+        cmd_output('git', 'switch', 'master')
+        install(C.CONFIG_FILE, store, hook_types=['post-merge'])
+        retc, stdout, stderr = cmd_output('git', 'merge', 'branch')
```

Comment:
this needs to use the special helper that the others use, otherwise it will write to the programmer's home directory under test: `cmd_output_mocked_pre_commit_home`